README.md

```diff
@@ -254,6 +269,35 @@ async function main () {
 main().catch(console.error);
 ```
 
+### Service account credentials
+
+Service accounts allow you to perform server to server, app-level authentication using a robot account.  You will create a service account, download a keyfile, and use that to authenticate to Google APIs.  To create a service account:
+- Go to the [Create Service Account Key page](https://console.cloud.google.com/apis/credentials/serviceaccountkey)
+- Select `New Service Account` in the drop down
+- Click the `Create` button
+
+Save the service account credential file somewhere safe, and *do not check this file into source control*!  To reference the service account credential file, you have a few options.
+
+#### Using the `GOOGLE_APPLICATION_CREDENTIALS env var
```

Comment:
Back tick is not closed here and the formatting is screwed up

Reply:
Sick catch